src/models/converters/transactions/tests/mod.rs

```diff
@@ -1,8 +1,8 @@
+pub(super) mod check_sender_or_receiver;
 mod data_size_calculation;
-mod summary;
 mod details;
-pub(super) mod transaction_types;
 pub(super) mod map_status;
 pub(super) mod missing_signers;
-pub(super) mod check_sender_or_receiver;
-pub(super) mod transfer_type_checks;
\ No newline at end of file
+mod summary;
```

Comment:
Wow this is weird, that the mod visibility has no impact on sorting ... would look nicer if they did :P